test/integration/golden_response/search_statvar/crime.json

```diff
@@ -55,31 +55,31 @@
   ],
   "statVarGroups": [
     {
-      "name": "CriminalActivities by Crime Type",
+      "name": "CriminalActivities With Crime Type",
```

Comment:
maybe use "by" in this case?

Reply:
Yeah, I was half wondering about that... ok, I'll make a change.